app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -1047,7 +1143,7 @@ public boolean openZimFile(File file, boolean clearHistory) {
 
   @Override
   public void onRequestPermissionsResult(int requestCode,
-      String permissions[], int[] grantResults) {
+                                         String permissions[], int[] grantResults) {
```

Comment:
Dont think this spacing is right?

Reply:
@mhutti1 It automatically does this in Android Studio, guess it follows some other style.

Reply:
Can you fix it then?